### ISSUE TITLE
Datagrid rows changing information is not read by screen reader after filtered

### ISSUE THREAD

**Issue by KingMario** [vmware/clarity#6499]
## Describe the bug

When the rows are filtered out, the rows changing information including total item count and/or rows on the current page is not read by screen reader

## How to reproduce

1.  Go to '[stackblitz clarity datagrid simple sample](https://stackblitz.com/edit/clarity-light-theme-v5-datagrid-simple)'
2.  Run a screen reader such as VoiceOver for MacOS or NVDA for Windows
3.  Click on the filter icon of datagrid 'name' column header and enter `ce` to filter out the rows
4.  The rows changed from 7 to 4 (including the header and footer), but nothing read by the screen reader

## Expected behavior

The rows changing information should be read by the screen reader. The information should include not only how many rows are shown in the current page, but also totally how many items are filtered out.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**

Angular 11

**Comment by gnomeontherun** [comment]
Hi KingMario, is this from a VMware product or something else?  This is probably duplicated in our list @scroniser may know.

**Comment by KingMario** [comment]
Hi Jeremy, it's from a portal of VMware, and is originally raised by India VPAT test team.

In the portal, I have created a directive to announce the info of sorting and filtering.

But I think it's better to have the information announced by the Datagrid component.

**Comment by scroniser** [comment]
Hi @KingMario! I’m on the internal accessibility team at VMware, specifically focused on assisting the Clarity team. Although I’m in the US, both locales fall under the same group. 

Given this request was generated from an internal audit, could you DM me on Slack so I can better assist?

**Comment by gnomeontherun** [comment]
Thanks @KingMario let's track and tackle these through our internal process so we don't have duplication of efforts here. I'm going to close this as a duplicate. Thanks!

Also, Clarity intentionally does not make announcements due to the complexity of trying to handle that automatically for each use case. We expect applications to handle this as necessary.
